turbinia/turbiniactl.py

```diff
@@ -805,19 +815,19 @@ def process_evidence(
     client(TurbiniaClient): TurbiniaClient used for creating requests.
     group_id(str): Group ID used for bulk processing.
     args(Namespace): commandline args.
-    browser_type(list(str)): List of browser types used for hindsight.
-    disk_name(list(str)): List of disk names used for processing cloud evidence.
-    embedded_path(list(str)): List of embedded paths for clouddiskembedded.
+    browser_type(str): List of browser types used for hindsight.
```

Comment:
The descriptions for most of these still refer to `List`s.

Reply:
Fixed